TrackingTools/DetLayers/interface/simple_stat.h

```diff
@@ -1,3 +1,7 @@
+#ifndef SIMPLE_STAT_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h